audify/utils/api_config.py

```
@@ -54,6 +70,551 @@ def speech_url(self) -> str:
+    def _get_client(self):
+        """Get or create Google TTS client."""
+        if self._client is None:
+            try:
+                from google.cloud import texttospeech
+
+                self._client = texttospeech.TextToSpeechClient()
+            except ImportError:
+                logger.error(
+                    "google-cloud-texttospeech is required. "
+                    "Install with: pip install google-cloud-texttospeech"
+                )
```

Ruff (0.14.14)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)